embedding-sdk/unauthenticated/src/index.js

```diff
@@ -0,0 +1,100 @@
+import "regenerator-runtime/runtime";
+import $ from "jquery";
+import ChartsEmbedSDK from "@mongodb-js/charts-embed-dom";
+
+const sdk = new ChartsEmbedSDK({
+  baseUrl: "https://charts.mongodb.com/charts-tomh-yfpqi" // Optional: ~REPLACE~ with the Base URL from your Embed Chart dialog
+});
+
+const chart = sdk.createChart({
+  chartId: "92f7af7d-de64-4772-9bf4-b6fd2f59ac59", // Optional: ~REPLACE~ with the Chart ID from your Embed Chart dialog
+  height: "700px"
+});
+
+async function renderChart() {
+  await chart.render(document.getElementById("chart"));
+
+  /*
+    chart.refresh()
+    Manually fetches the latest data for the chart
+   */
+  $("#refresh").on("click", () => {
+    chart.refresh();
+  });
+
+  /*
+    chart.setRefreshInterval(interval: number)
+    The default rate is to never refresh after rendering.
+    The minimum rate is every 10 seconds.
+    Setting to 0 returns the chart to the default setting.
+
+    chart.getRefreshInterval()
+    Returns a number pertaining to the charts current
+    refresh interval.
+   */
+  $("#refresh-interval").on("change", async e => {
+    var refreshInterval = e.target.value;
+    refreshInterval
+      ? chart.setRefreshInterval(Number(refreshInterval))
+      : chart.setRefreshInterval(0);
+
+    var currentRefreshInterval = await chart.getRefreshInterval();
+    $("#currentRefreshInterval").text(currentRefreshInterval);
+  });
+
+  /*
+    chart.setFilter(filter: object)
+    Filters the chart with the given filter string.
+    It's important to manually whitelist fields you want users
+    to be able to filter on. Do this in the Embedded settings of 
+    your chart. 
+
+    chart.getFilter()
+    Returns the current filter object. The key is the field,
+    and the value the query.
+   */
+  $("#country-filter").on("change", async e => {
+    const country = e.target.value;
+    country
+      ? chart.setFilter({ "address.country": country })
+      : chart.setFilter({});
+
+    var currentFilter = await chart.getFilter();
+    var keyName = Object.keys(currentFilter)[0];
```

Comment:
Can't I have a filter with more than one key it it?

Reply:
if you're expecting it to be 'address.country', you might as well hardcode it below:
```
country
      ? $("#currentFilter").text(
          "{ " + 'address.country' + " : " + currentFilter['address.country'] + " }"
        )
      : $("#currentFilter").text("{ }");
  });
```

Reply:
It might simplify this area if we just have one big IF-ELSE for the different cases. i.e.
```
if (country) {
  await chart.setFilter({ "address.country": country })
  const filter = await chart.getFilter();
  $("#currentFilter").text(JSON.stringify(filter));
} else {
  await chart.setFilter({})
  const filter = await chart.getFilter();
  $("#currentFilter").text(JSON.stringify(filter));
}
```

Reply:
Good point, this is a bit hacky.

Reply:
I can't say I agree with hard coding the filter, since we encourage users to replace the sample data in the Readme, with some notes on getting the filtering to work. I think my assumption that there would only be one filter is better than that of hard coding the query string. Thoughts?